R/learner_mboost_surv_glmboost.R

```diff
@@ -47,7 +57,7 @@ LearnerSurvGLMBoost = R6Class("LearnerSurvGLMBoost",
         contrasts.arg = p_uty(tags = "train")
       )
 
-      ps$values = list(family = "coxph")
+      ps$values = list(family = "coxph", center = FALSE)
```

Comment:
why was this changed? Such changes must always be documented in the learner description and we should give an explanation why this is the case

Reply:
if it is deprecated as you write below we should maybe just remove the parameter, but maybe I don't understand it properly yet.

Reply:
Went through a bit more deeply in the documentation, it's actually `center` = `TRUE` by default:
![image](https://github.com/user-attachments/assets/9707d759-5f5c-49c4-9ba5-21b8f74a5ec8)

Reply:
the other thing is that `center` it is part of the control variables but deprecated but they also have it right now in the main `glmboost` function (but since it is still accessible from the `mboost_control` so we need to remove it from there):
![image](https://github.com/user-attachments/assets/8c6c3594-c4c9-4699-a770-77c3fb446114)

Reply:
@sebffischer summary: it's a code-related thing and I needed to do that since they haven't completely deprecated `center` from the `mboost::mboost_control()` function (it's there + in the main function as parameter).

Also, I reverted to `center = TRUE` as it was before

Reply:
ok great!